scripts/check_type_hint_coverage.py

```diff
@@ -4,7 +4,7 @@
 from typing import Dict, List, Optional
 
 TYPE_HINT_ERROR_THRESHOLD: int = (
-    1000  # This number is to be reduced as we annotate more functions!
+    1500  # This number is to be reduced as we annotate more functions!
```

Comment:
Can we update this comment to explain when this number may need to be increased? I don't follow why based on the changes in `pyproject.toml`.

Reply:
@NathanFarmer This came from somewhere else (I did not modify this, as far as I can recall).  Thanks.

Reply:
@alexsherstinsky is it possible to modify it back to 1000 then?

Reply:
I made this change.
`1000` will cause the CI to fail.
The changes to what is being excluded/inlcuded in the mypy config, break the way this script used to operate.